From b23df206777642fc0a58cdfce9ddde0417b6ea7b Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Fri, 3 Feb 2023 08:55:36 +0100
Subject: [PATCH 0069/1141] arm64: dts: st: enable secure arm-wdt watchdog on
 stm32mp257f-ev

Enable the watchdog and define the default timeout to 32 seconds.

Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Ic418ff54c4a58d46084ffcbd9ad16ea9dfa9d972
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/288995
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
index c29d2baeb0ed..e402e5244031 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
@@ -28,3 +28,8 @@ memory@80000000 {
 		reg = <0x0 0x80000000 0x1 0x0>;
 	};
 };
+
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
-- 
2.39.2

